integrated_channels/integrated_channel/tasks.py

```diff
@@ -73,110 +104,102 @@ def transmit_learner_data(username, channel_code, channel_pk):
     start = time.time()
     api_user = User.objects.get(username=username)
     integrated_channel = INTEGRATED_CHANNEL_CHOICES[channel_code].objects.get(pk=channel_pk)
-    generate_formatted_log(
-        'Batch processing learners for integrated channel. Configuration: {configuration}'.format(
-            configuration=integrated_channel,
-        ),
-        channel_name=channel_code,
-        enterprise_customer_identifier=api_user.username
-    )
+    _log_batch_task_start('transmit_learner_data', channel_code, api_user.id, integrated_channel)
 
-    # Note: learner data transmission code paths don't raise any uncaught exception, so we don't need a broad
-    # try-except block here.
+    # Note: learner data transmission code paths don't raise any uncaught exception,
+    # so we don't need a broad try-except block here.
     integrated_channel.transmit_learner_data(api_user)
 
     duration = time.time() - start
-    generate_formatted_log(
-        'Batch learner data transmission task finished. Configuration: {configuration},'
-        ' Duration: {duration}'.format(
-            configuration=integrated_channel,
-            duration=duration
-        ),
-        channel_name=channel_code,
-        enterprise_customer_identifier=api_user.username
-    )
+    _log_batch_task_finish('transmit_learner_data', channel_code, api_user.id, integrated_channel, duration)
 
 
 @shared_task
 @set_code_owner_attribute
-def transmit_single_learner_data(username, course_run_id):
+def transmit_single_learner_data(learner_username, course_run_id):
     """
-    Task to send single learner data to each linked integrated channel.
```

Comment:
given this is the `single_learner` method and I do think we have a bulk one somewhere, is there a reason to remove "single" here?

Reply:
reverting